nmdc_server/crud.py

```diff
@@ -475,10 +475,15 @@ def get_zip_download(db: Session, id: UUID) -> Optional[str]:
             logger.warning(f"Data object url for {file.path} was {data_object.url}")
             continue
 
+        if not data_object.file_size_bytes:
```

Comment:
I suggest documenting our rationale here.

```suggestion
        # Skip data objects whose `file_size_bytes` is `0` or `None`, since `mod_zip`
        # considers a file list containing any file sizes of "0" or "" to be invalid.
        if not data_object.file_size_bytes:
```